packages/Ludown/test/ludown.negative.test.suite.js

```diff
@@ -39,7 +39,7 @@ describe('Negative tests', function() {
     it('should show ERROR when no parser decorations are found in a line', function(done) {
         exec(`node ${ludown} parse toluis --in ${TEST_ROOT}/testcases/bad2.lu`, (error, stdout, stderr) => {
             try {
-                assert.equal(stderr.includes('not part of a Intent/ Entity/ QnA'), true);
+                assert.equal(stderr.includes("syntax error: mismatched input 'f' expecting"), true);
```

Comment:
We should make the error more human readable. Not sure what it means to be expecting 'f'

Reply:
The actual error message is "[ERROR] line 1:0 - line 1:1: syntax error: mismatched input 'f' expecting {<EOF>, MODEL_INFO, NEWLINE, QNA, HASH, DOLLAR, IMPORT_DESC}"